sdcm/cluster.py

```diff
@@ -283,6 +283,9 @@ def init(self) -> None:
         self._init_port_mapping()
 
         self.set_keep_alive()
+        if self.node_type == "db":
+            self.remoter.sudo('sh -c "echo \'kernel.perf_event_paranoid = 0\' >> /etc/sysctl.conf"', verbose=True)
```

Comment:
use `sdcm.remote.base.shell_script_cmd` instead of `sh -c`

Reply:
done